app/controllers/admin/attachments_controller.rb

```diff
@@ -100,6 +103,7 @@ def build_external_attachment
   def build_file_attachment
     FileAttachment.new(attachment_params).tap do |file_attachment|
       file_attachment.build_attachment_data unless file_attachment.attachment_data
+      file_attachment.attachment_data.attachable = attachable
```

Comment:
I think we might need to do something similar for `Admin::BulkUploadsController` which also appears to create attachments.

Reply:
@danielroseman mentioned that it's possible the `BulkUploadsController` was only used for transition and is no longer necessary. I plan to investigate further.